charts/camunda-platform/templates/camunda/_helpers.tpl

```diff
@@ -51,17 +51,16 @@ Define common labels, combining the match labels and transient labels, which mig
 {{- define "camundaPlatform.labels" -}}
 {{- template "camundaPlatform.matchLabels" . }}
 helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-{{- if .Values.image }}
-    {{- if .Values.image.tag }}
+  {{- if .Values.zeebe.image.tag }}
```

Comment:
It should be based on the component, not always `zeebe`.
We do something similar here:
https://github.com/camunda/camunda-platform-helm/blob/c4e4f9c08164fdd0d5c3bdfc9cc9282df38e7eb5/charts/camunda-platform/templates/camunda/_helpers.tpl#L126